playbook/app/pb_kits/playbook/pb_passphrase/_passphrase.tsx

```diff
@@ -15,15 +15,15 @@ import PbReactPopover from '../pb_popover/_popover'
 import TextInput from '../pb_text_input/_text_input'
 
 type PassphraseProps = {
-  aria?: object,
+  aria?: { [key: string]: string },
   confirmation?: boolean,
   className?: string,
   data?: object,
   dark?: boolean,
   id?: string,
   inputProps?: {},
   label?: string,
-  onChange: (String) => void,
+  onChange: (arg0: String) => void,
```

Comment:
Let's avoid use `arg0` if possible, and use something more expressive. 
I like that we are improving the readability and expressiveness of our code by being more explicit, but let's try to really name this a little better. 

What about `inputValue`? I know it's just semantics, but for some reason the arg0 to me looks a little messy.

Reply:
Great catch. I will change it. Thank you @jasperfurniss